frontend/src/views/activity/Activity.vue

```diff
@@ -99,24 +99,127 @@ Displays a single activity
+                          </v-btn>
+                        </template>
+                        <span>
+                          {{ $tc('views.activity.activity.copyScheduleEntryDayBefore') }}
+                        </span>
+                      </v-tooltip>
+                      <dialog-schedule-entry-create
+                        :camp="scheduleEntryItem.period().camp"
+                        :activity="scheduleEntryItem.activity"
+                        :schedule-entry="scheduleEntryItem">
+                        <template #activator="{on: dialog}">
+                          <v-tooltip bottom>
+                            <template #activator="{ on: tooltip, attrs }">
+                              <v-btn icon v-bind="attrs" v-on="{ ...tooltip, ...dialog}">
+                                <v-icon>
+                                  mdi-calendar-clock
+                                </v-icon>
+                              </v-btn>
+                            </template>
+                            <span>
+                              {{ $tc('views.activity.activity.copyScheduleEntryAnyTime') }}
+                            </span>
+                          </v-tooltip>
+                        </template>
+                      </dialog-schedule-entry-create>
+                      <v-tooltip bottom>
+                        <template #activator="{ on, attrs }">
+                          <v-btn :disabled="!scheduleEntryItem.copyToDayAfter"
+                                 icon
+                                 v-bind="attrs"
+                                 v-on="on"
+                                 @click="duplicateScheduleEntry(scheduleEntryItem, 24*60)">
+                            <v-icon>
+                              mdi-calendar-arrow-right
+                            </v-icon>
+                          </v-btn>
+                        </template>
+                        <span>
+                          {{ $tc('views.activity.activity.copyScheduleEntryDayAfter') }}
+                        </span>
+                      </v-tooltip>
+                    </v-card>
+                  </v-menu>
+                  <dialog-entity-delete :entity="scheduleEntryItem">
+                    <template #activator="{ on: dialog }">
+                      <v-tooltip bottom>
+                        <template #activator="{ on: tooltip }">
+                          <v-icon v-on="{ ...tooltip, ...dialog}">mdi-calendar-minus</v-icon>
+                        </template>
+                        {{ $tc('views.activity.activity.deleteScheduleEntry') }}
+                      </v-tooltip>
+                    </template>
+                    ScheduleEntry wirklich löschen?
+                    <ul>
+                      <li>
+                        ({{ scheduleEntryItem.number }})
+                        {{ $date.utc(scheduleEntryItem.startTime).format($tc('global.datetime.dateShort')) }} <b>
```

Comment:
Now i see this logic the third time duplicated.
I will extract this into a component.
Extracting this into a component doesn't belong into this PR, i will do that.

Reply:
thx